docs/webnn.md

```diff
@@ -0,0 +1,301 @@
+
+WebNN is designed as a backend API for web frameworks. For Windows, we recommend using [ONNX Runtime Web](https://onnxruntime.ai/docs/tutorials/web/). This gives a familiar experience to using DirectML and ONNX Runtime natively so you can have a consistent experience deploying AI in ONNX format across web and native applications.
+
+## WebNN requirements
+
+You can check information about your browser by navigating to about://version in your chromium browser's address bar.
+
+| Hardware | Web Browsers | Windows version | ONNX Runtime Web version | Driver Version |
+| --- | --- | --- | --- | --- | 
+| **GPU** | WebNN requires a Chromium browser*. Please use the most recent version of Microsoft Edge Beta. | Minimum version: Windows 11, version 21H2. |Minimum version: 1.18 | Install the latest driver for your hardware. | 
+
+![Diagram of the structure behind integrating WebNN into your web app](images/webnn-diagram.png)
+
+> [!NOTE]
+> Chromium based browsers can currently support WebNN, but will depend on the individual browser's implementation status.
+
+## Model support 
+
+### GPU (Preview):
+When running on GPUs, WebNN currently supports the following models:
+
+* [Stable Diffusion Turbo](https://microsoft.github.io/webnn-developer-preview/demos/sd-turbo/)
+* [Stable Diffusion 1.5](https://microsoft.github.io/webnn-developer-preview/demos/stable-diffusion-1.5/)
+* [Whisper-base](https://microsoft.github.io/webnn-developer-preview/demos/whisper-base/)
+* [MobileNetv2](https://microsoft.github.io/webnn-developer-preview/demos/image-classification/)
+* [Segment Anything](https://microsoft.github.io/webnn-developer-preview/demos/segment-anything/)
+* [ResNet](https://microsoft.github.io/webnn-developer-preview/demos/image-classification/?provider=webnn&devicetype=gpu&model=resnet-50&run=5)
+* [EfficientNet](https://microsoft.github.io/webnn-developer-preview/demos/image-classification/?provider=webnn&devicetype=gpu&model=efficientnet-lite4&run=5)
+* SqueezeNet 
+
+WebNN also works with custom models as long as operator support is sufficient. Check status of operators [here](https://webmachinelearning.github.io/webnn-status/).
+
+## FAQ
+
+#### **How do I file an issue with WebNN?**
+
+For general issues with WebNN, please file an issue on our [WebNN Developer Preview GitHub](https://github.com/microsoft/webnn-developer-preview/issues)
+
+For issues with ONNX Runtime Web or the WebNN Execution Provider, go to the [ONNXRuntime Github](https://github.com/microsoft/onnxruntime/issues).
+
+#### **How do I debug issues with WebNN?**
+
+The [WebNN W3C Spec](https://www.w3.org/TR/webnn/) has information on error propagation, typically through DOM exceptions. The log at the end of about://gpu may also have helpful information. For further issues please file an issue as linked above.
+
+#### Does WebNN support other operating systems?
+
+Currently, WebNN best supports the Windows operating system. A version for Mac operating systems is in progress.
```

Comment:
More than just Mac are being written, including ChromeOS, and Android.  

I think it's more accurate for the second sentence to simply say "Versions for other operating systems are in progress".